docker-app/qfieldcloud/core/tests/test_secret_packages.py

```diff
@@ -55,6 +59,20 @@ def setUp(self):
 
         self.p1.direct_collaborators.bulk_create(collaborators)
 
+        self.geodb, _ = Geodb.objects.get_or_create(
```

Comment:
```suggestion
        self.geodb = Geodb.objects.create(
```

Why `get_or_create`?

Reply:
Why not ? Not.

Reply:
Because you are assuming the presense of a database which should not be the case. For more deterministic tests, I would suggest we always create a database and use it instead of relying on the order and combination of the tests ran before, which is most probably the reason to have a test db already created.

Reply:
Fully agree.